extensions/persist/pom.xml

```diff
@@ -27,6 +27,12 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
+      <scope>compile</scope>
```

Comment:
This makes JUnit a runtime dependency of `guice-persist`, which is probably not what most people want.